### PR TITLE
Add ignore_timestamps_error to OpenEphysLegacyRecording

### LINKED ISSUE #2107: How to sort only a segment of an Open Ephys recording
Hello

I would appreciate any advice on how to sort only a segment of an open ephys recording. A portion of my recordings have a discontinuity in the timestamps which produces an error like #1194 . I've found that this error is caused by a short ~17 ms discontinuity ~34 ms before the end of the recording. The recording time stamps also do not start at 0, but I have other recordings that also do not start at zero and spike sort successfully (so long as they do not possess the discontinuity at the end).

I attempted to use the ignore_timestamps_errors option described in https://github.com/NeuralEnsemble/python-neo/issues/1210 and https://github.com/NeuralEnsemble/python-neo/pull/1213, but this did not work. I received a syntax error, but perhaps I am not using the most up to date version of the code. I'd rather just cut off the end of the recording in case there are other discontinuities in the middle of the recording that I should be aware of.

Is there a way in spike interface to only extract and sort a certain part of the recording, such as from time step 1 to time step end - 30000? I believe this would occur at:

raw_rec = si.read_openephys(openephys_folder,stream_name='Signals CH')

I investigated the block_index argument, but I'm not sure of the correct syntax to use. I would appreciate any advice that people could provide. I've looked around the issues and documentation and I couldn't find anything that directly addressed this point, but if I missed it, I apologize and please direct me to the correct resource. Thank you!

~Nick

### PR DESCRIPTION
**Pull request by alejoe91**
Should fix #2107

### CODE REVIEW

File: src/spikeinterface/extractors/neoextractors/openephys.py [hunk 1]
```diff
@@ -82,7 +82,9 @@ def __init__(
             all_annotations=all_annotations,
             **neo_kwargs,
         )
-        self._kwargs.update(dict(folder_path=str(Path(folder_path).absolute())))
+        self._kwargs.update(
+            dict(folder_path=str(Path(folder_path).absolute())), ignore_timestamps_errors=ignore_timestamps_errors
```
Comment (samuelgarcia) [inline src/spikeinterface/extractors/neoextractors/openephys.py:86]: This option is very very dangerous because it do not ensure channel alignement!

Comment (alejoe91) [reply src/spikeinterface/extractors/neoextractors/openephys.py:86]: unrelated though. If it's an argument, it should be in the `self._kwargs`. The misalignment problem is NEO-related

**Review by h-mayorquin — APPROVED** [review]
Is there anything stoping this?

By the way, I think we should use `resolve` instead of `absolute`:

https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve

**Comment by h-mayorquin** [comment]
@samuelgarcia 
Thanks for exlpaining Sam. I guess you and @alejoe91 discussed this previously but I did not understand that from the discussion above. You don't want to propagate the option until is fixed on neo.Got you.

**Comment by alejoe91** [comment]
Closing this because we need a proper fix on the NEO side. See this issue: https://github.com/NeuralEnsemble/python-neo/issues/1351

**Pull request closed without merging**